pyramid_nacl_session/scripts.py

```diff
@@ -27,22 +27,23 @@ def print_secret():  # pragma: NO COVER
     The secret is a 32-byte random string, encoded using
     :func:`binascii.hexlify`.
     """
+
```

Comment:
Janitorial changes (e.g., blackening) should normally be done in a separate PR:  they make reviewing substantive changes much more difficult.

Reply:
Ok sorry, I'll think about it next time..!